Week-1/index.html

```diff
@@ -0,0 +1,185 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <style>
```

Comment:
Instead of adding css here itself , try to write css in a separate file and then import here

Reply:
I have created a separate stylesheet and moved all the css into style.css file.